src/bootstrap/src/ferrocene/code_coverage.rs

```diff
@@ -124,8 +124,11 @@ pub(crate) fn generate_coverage_report(builder: &Builder<'_>) {
             "ferrocene/library/backtrace-rs",
             "ferrocene/library/libc",
             "library/alloc",
+            "library/compiler-builtins",
+            "library/coretests",
```

Comment:
We found these were helpful to have in the `.info` file while debugging coverage issues, I'm not entirely sure this is a desirable change for that reason...

Reply:
When @pvdrz and I were debugging coverage we found we looked for tests in here to validate the CI did in fact run them and instrument them.